README.md

```diff
@@ -1,6 +1,19 @@
 tuxmea-backupexec
 =================
 
-Pupept module for Symantec BackupExec agent
+Puppet module for Symantec BackupExec agent. Should work on RedHat and Debian based systems. 
+Currently vrtsralus v20.0.1188.1863 is used.
 
-Currently vrtsralus v20.0.1188.1863 is used as unix client on debian
+Requirements:
+ 1. ``` vrtsralus ``` installable via repository or manuall installed (not available in a public repository)
+ 2. defined variables in ~/hieradata/common.yaml (```backupexec::be_server``` and ```backupexec::be_userpw```)
```

Comment:
for single words/strings you can use single quotes instead of three at the beginning and three at the end.